parking_permits_app/reversion.py

```diff
@@ -53,7 +53,7 @@ def change_message(self):
                 and target_model.objects.filter(pk=self.new_value).first()
             )
             return f"{self.field.verbose_name}: {old_instance} --> {new_instance}"
-        elif isinstance(self.field, models.DateTimeField):
+        elif isinstance(self.field, models.DateTimeField) and self.old_value:
```

Comment:
I think we want to record the change even the old_value is None, but you are right that we need to check that the value exists before doing `strftime` on them. So I think it's the next two lines that should be changed.